benchmarks/TFLite/CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ set(MOBILENET_V3SMALL_MODULE
 ################################################################################
 # Add benchmarks for all platforms.                                            #
 ################################################################################
+include(linux-riscv.cmake)
```

Comment:
We'd want to move the configurations for mobile CPU/GPU in the above to its separate files and include here to be consistent. Don't need to happen in this patch though.